src/libraries/Microsoft.CSharp/src/MultilingualResources/Microsoft.CSharp.de.xlf

```diff
@@ -7,353 +8,353 @@
       <group id="MICROSOFT.CSHARP/RESOURCES/STRINGS.RESX" datatype="resx">
         <trans-unit id="InternalCompilerError" translate="yes" xml:space="preserve">
           <source>An unexpected exception occurred while binding a dynamic operation</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">Unerwartete Ausnahme beim Binden eines dynamischen Vorgangs</target>
+          <target state="translated" state-qualifier="tm-suggestion">Unerwartete Ausnahme beim Binden eines dynamischen Vorgangs</target>
```

Comment:
@jonfortescue @cristianosuzuki77 this looks to have resolved the lost translation problem, thank you!

Reply:
glad to see it!